### DISCUSSION TITLE
What is `did_full = did_full` doing in this code?

### DISCUSSION THREAD

**Discussion by rdimaio** [rucio/rucio#6678] [Development Q&A] [answered]
The lines I am confused about are 169 and 184 below:

https://github.com/rucio/rucio/blob/339744866e9550f0f068f8e3c9574a95ccaa7d59/lib/rucio/core/did_meta_plugins/mongo_meta.py#L164-L187

The statement is the following (it's the same on both lines 169 and 184):

```python
did_full = did_full = "{}:{}".format(did['scope'], did['name']) 
```

I don't understand the `did_full = did_full` part of this. Is it a typo or is it needed?

**Comment by bari12** [comment]
I think this is a typo. @robbarnsley ?

**Reply by robbarnsley** [reply, marked as answer]
Looks like I did a sloppy job of copy/paste. Typo!

**Reply by rdimaio** [reply]
Thanks - will make a quick PR to fix it

Edit: PR: https://github.com/rucio/rucio/pull/6711
